src/packages/react-prisma/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "react-prisma",
+  "version": "0.0.5",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "license": "Apache-2.0",
+  "author": "Tim Suchanek <suchanek@prisma.io>",
+  "devDependencies": {
+    "@types/jest": "26.0.19",
+    "@types/node": "12.19.11",
+    "@typescript-eslint/eslint-plugin": "4.10.0",
+    "@typescript-eslint/parser": "4.10.0",
+    "eslint": "7.15.0",
+    "eslint-config-prettier": "7.0.0",
+    "eslint-plugin-eslint-comments": "3.2.0",
+    "eslint-plugin-jest": "24.1.3",
+    "eslint-plugin-prettier": "3.3.0",
+    "jest": "26.6.3",
+    "lint-staged": "10.5.3",
+    "prettier": "2.2.1",
+    "strip-ansi": "6.0.0",
+    "ts-jest": "26.4.4",
+    "typescript": "4.1.3",
+    "react": "^17.0.0",
+    "@prisma/client": "*"
+  },
+  "scripts": {
+    "build": "tsc -d",
+    "prepublishOnly": "pnpm run build",
+    "format": "prettier --write .",
+    "lint": "eslint --cache --fix --ext .ts .",
+    "lint-ci": "eslint --ext .ts .",
+    "test": "jest",
+    "precommit": "lint-staged"
+  },
+  "files": [
+    "!**/__tests__",
+    "dist"
+  ],
+  "lint-staged": {
+    "*.ts": [
+      "eslint",
+      "prettier --write"
+    ]
+  },
+  "peerDependencies": {
+    "react": "^17.0.0",
+    "@prisma/client": "*"
```

Comment:
maybe >=2.13.0 or something like that?

Reply:
There is no constraint necessary, as we didn't need to change anything in the client.

Reply:
But it relies on dmmf stuff so it's likely broken in older versions I guess

Reply:
The DMMF has always been exported like that.

Reply:
Ok, I would at least put >=2.0.0 especially with the prisma npm switch if it happens though.